packages/manager/modules/ovh-payment-method/rollup.config.js

```diff
@@ -0,0 +1,33 @@
+import path from 'path';
+import rollupConfig from '@ovh-ux/component-rollup-config';
+
+const config = rollupConfig(
+  {
+    input: './src/index.js',
+  },
+  {
+    lessTildeImporter: {
+      paths: [
+        path.resolve(__dirname, 'node_modules'),
+        path.resolve(__dirname, '../../node_modules'),
+      ],
+    },
+  },
+);
+
+const outputs = [config.es()];
+
+// if (process.env.BUILD === 'production') {
+//   outputs.push(config.cjs());
+//   outputs.push(
+//     config.umd({
+//       output: {
+//         globals: {
+//           angular: 'angular',
+//         },
+//       },
+//     }),
+//   );
+// }
```

Comment:
remove this comment?

Reply:
Mmmmh. I don't know yet >_< As this module should be moved in components folder as it should be public in order to be used elsewhere of the manager monorepository.
Sorry, I should have left a comment when creating the PR